plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -193,13 +197,15 @@ static void dump_source_info(struct ffmpeg_source *s, const char *input,
 			"\tinput_format:            %s\n"
 			"\tis_looping:              %s\n"
 			"\tis_hw_decoding:          %s\n"
+			"\tis_not_seekable          %s\n"
```

Comment:
Minor nitpick:  Place a colon (`:`) after the text label `is_not_seekable`.  Don't forget to remove a space between the label and the data to keep everything aligned.